.gitignore

```diff
@@ -29,3 +29,4 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+.idea/
```

Comment:
I see that `.idea/ruby-todo.iml` is tracked and commited in this PR.
Here is stackoverflow for how to removed ignored files from tracking: https://stackoverflow.com/a/1274447/18171185